### PR TITLE
Add release notes for 2.13.8

### PR DESCRIPTION
**Pull request by bunchesofdonald**

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,98 @@
...
+        name="my-first-deployment",
+        tags=["onboarding"],
+        parameters={"goodbye": True},
+        interval=60,
+        work_pool_name="above-ground",
+        image='my_registry/hello_world:demo'
+    )
+```
+
+You can also use `deploy` as a replacement for `serve` if you want to deploy multiple flows all at once.
```
Comment (discdiver) [inline RELEASE-NOTES.md:61]:
```suggestion
You can also use `deploy` as a replacement for `serve` if you want to deploy multiple flows at once.
```

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,98 @@
 # Prefect Release Notes
 
+## Release 2.13.8
+
+### Introducing `flow.deploy`
+
+When we released `flow.serve`, we introduced a radically simple way to deploy flows. Serving flows is perfect for many use cases, but the need for persistent infrastructure means serving flows may not work well for flows that require expensive or limited infrastructure to execute.
```
Comment (discdiver) [inline RELEASE-NOTES.md:7]:
```suggestion
When we released `flow.serve`, we introduced a radically simple way to deploy flows. Serving flows is perfect for many use cases, but the need for persistent infrastructure means serving flows may not work well for flows that require expensive or limited infrastructure.
```

**Review by discdiver — APPROVED** [review]
Nice! I suggested cutting a few extra words, but not a big deal.
I assume most folks will go to the tutorial to actually run, but if not, maybe worth noting that they will need to have Docker running to build the image (and be authenticated to DockerHub if pushing the image).

Could also note that they can bring their own Docker image, but depends how in the weeds you want to get.

**Review by zhen0 — APPROVED** [review]
LGTM!

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,5 +1,98 @@
 # Prefect Release Notes
 
+## Release 2.13.8
+
+### Introducing `flow.deploy`
+
+When we released `flow.serve`, we introduced a radically simple way to deploy flows. Serving flows is perfect for many use cases, but the need for persistent infrastructure means serving flows may not work well for flows that require expensive or limited infrastructure.
+
+We're excited to introduce `flow.deploy` as a simple transition from running your served flows on persistent infrastructure to executing your flows on dynamically provisioned infrastructure via work pools and workers. `flow.deploy` ensures your flows execute consistently across environments by packaging your flow into a Docker image and making that image available to your workers when executing your flow.
+
```
Comment (serinamarie) [inline RELEASE-NOTES.md:10]: Seeing this as a user of prefect deploy, maybe mention that this is the Python-based deployment experience companion to prefect deploy?

Comment (bunchesofdonald) [reply RELEASE-NOTES.md:10]: Something like:

```We're excited to introduce `flow.deploy`, the Python-based deployment experience companion to `prefect deploy`. A simple transition from running your served flows on persistent infrastructure to executing your flows on dynamically provisioned infrastructure via work pools and workers. `flow.deploy` ensures your flows execute consistently across environments by packaging your flow into a Docker image and making that image available to your workers when executing your flow.
``` ?

Comment (serinamarie) [reply RELEASE-NOTES.md:10]: hm, sorry, I think it's a companion to `flow.serve`. I'm not sure what the right word is

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -1,5 +1,98 @@
...
+### Enhancements
+- Add `last_polled` column to deployment table — https://github.com/PrefectHQ/prefect/pull/10949
+- Add `status` and `last_polled` to deployment API responses — https://github.com/PrefectHQ/prefect/pull/10951
+- Add flow run graph v2 endpoint tuned for UI applications — https://github.com/PrefectHQ/prefect/pull/10912
+- Add deployment status tests for `/work_pools/{name}/get_scheduled_flow_runs` and `/work_queues/{id}/get_runs` endpoints — https://github.com/PrefectHQ/prefect/pull/10973
+- Update `/deployments/get_scheduled_flow_runs` endpoint to update deployment status — https://github.com/PrefectHQ/prefect/pull/10969
+
+### Fixes
+- Clarify CLI prompt message for missing integration library for worker — https://github.com/PrefectHQ/prefect/pull/10990
+- Renamed ruamel-yaml to ruamel.yaml in requirements.txt — https://github.com/PrefectHQ/prefect/pull/10987
```
Comment (serinamarie) [inline RELEASE-NOTES.md:79]:
```suggestion
- Renamed `ruamel-yaml` to `ruamel.yaml` in `requirements.txt` — https://github.com/PrefectHQ/prefect/pull/10987
```

File: RELEASE-NOTES.md [hunk 5]
```diff
@@ -1,5 +1,98 @@
...
+- Make work pool banner copy less confusing — https://github.com/PrefectHQ/prefect/pull/10992
```
Comment (serinamarie) [inline RELEASE-NOTES.md:80]:
```suggestion
- Clarify work pool banner on Work Pool page UI — https://github.com/PrefectHQ/prefect/pull/10992
```

File: RELEASE-NOTES.md (hunk 3 above)
Comment (bunchesofdonald) [reply RELEASE-NOTES.md:10]: Yeah, I was a bit confused. Maybe just literally '... introduce `flow.deploy`, a companion to `flow.serve`, a simple ...`

Comment (bunchesofdonald) [reply RELEASE-NOTES.md:10]: @desertaxle ^^ thougths?

Comment (desertaxle) [reply RELEASE-NOTES.md:10]: I'm not sure how to mention `prefect deploy` here. The message would be something along the lines of 'you can keep using `prefect deploy`', but I'm not sure if explicitly saying that will take the focus off of `flow.deploy`. 

I lean towards not mentioning `prefect deploy` here unless someone else feels strongly that we should.

Comment (bunchesofdonald) [reply RELEASE-NOTES.md:10]: That works for me.

**Pull request merged**